doc/source/getting_started_with_Shelly.rst

```diff
@@ -0,0 +1,82 @@
+===================================
+Getting started with Shelly(Linux)
```

Comment:
Guide for Shelly?

Reply:
Other docs had similar heading so I chose to go with the same too. Do you need me to change it to "Guide for installing Araali Shelly" ?

Reply:
Getting started is literally getting started with araali (on eks, gke, rafay, ...). You are on one of these platforms and now you want to get started with araali

Shelly, you are not really getting started with araali. Either you are already on araali, or you don't want to (and we don't require it either)

It's more of a guide to running Araali Shelly (araali protected or not)

Reply:
Also you don't get started. Araali is a journey, you get started and stay with it

Shelly is a one time assessment. You run and get done right away. You didn't get started. You did a point in time assessment. You got finished too.

You could say: Test defense readiness with Araali Shelly. Or readiness assessment with Araali Shelly